backend/src/main/java/site/coduo/common/controller/CommonErrorController.java

```diff
@@ -0,0 +1,19 @@
+package site.coduo.common.controller;
+
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.RestControllerAdvice;
+import org.springframework.web.servlet.resource.NoResourceFoundException;
+
+import site.coduo.common.controller.error.CommonApiError;
+import site.coduo.common.controller.response.ApiErrorResponse;
+
+@RestControllerAdvice
+public class CommonErrorController {
+
+    @ExceptionHandler
```

Comment:
```suggestion
    @ExceptionHandler(NoResourceFoundException.class)
```
페어룸 익셉션 핸들러에는 예외 클래스 명시해줬습니다